content/how-to/iac-checks/gitlab/index.md

```diff
@@ -5,7 +5,7 @@ lead: "Configuring infrastructure as code on GitLab"
 date: 2022-01-12T02:48:57+00:00
 draft: false
 weight: 200
-toc: true
+toc: false
```

Comment:
Setting this across multiple pages so they don't appear in the table of contents.